lib/Models/CsvCatalogItem.js

```diff
@@ -553,6 +553,7 @@ CsvCatalogItem.prototype.setChartable = function() {
     tableStructure.allowMultiple = true;
     this.isMappable = false;
     tableStructure.getColorCallback = this.getNextColor.bind(this);
+    tableStructure.disableColumnsWithDifferentXType = this.disableColumnsWithDifferentXType.bind(this);
```

Comment:
This is valid JS but we always try to declare all possible properties in the constructor. So add `this.disableColumnsWithDifferentXType = undefined;` to `DisplayVariablesConcept` (which is the parent class of `TableStructure`).